README.md

```diff
@@ -108,8 +108,7 @@ used to invoke pip matches the interpreter used to run `py_binary` targets. By
 default, `pip_install` uses the system command `"python3"`. This can be overridden by passing the
 `python_interpreter` attribute or `python_interpreter_target` attribute to `pip_install`.
 
-You can have multiple `pip_install`s in the same workspace, e.g. for Python 2
```

Comment:
This was never true and shouldn't have been in docs. Python 2 couldn't be used with `pip_install`, it only ever supported Python 3.5+ I think.